Assets/Scripts/SS3D/Systems/Inventory/Containers/ContainerItemDisplay.cs

```diff
@@ -70,16 +70,16 @@ private void ContainerOnItemAttached(object sender, Item item)
                 // Create new (temporary) point
                 // HACK: Required because rotation pivot can be different
                 GameObject temporaryPoint = new GameObject("TempPivotPoint");
-                
+
                 temporaryPoint.transform.SetParent(attachedContainer.Displays[index].transform, false);
                 temporaryPoint.transform.localPosition = Vector3.zero;
-                temporaryPoint.transform.rotation = attachmentPoint.root.rotation *  attachmentPoint.localRotation;
-                
+
                 // Assign parent
                 itemTransform.SetParent(temporaryPoint.transform, false);
+                temporaryPoint.transform.rotation = attachmentPoint.root.rotation * attachmentPoint.localRotation;
```

Comment:
`attachmentPoint.root.rotation` seems very sketchy to me, as the root is not reliable to be things we want them to be. Maybe we can tweak this in the future